jwst/tests_nightly/general/pipelines/test_coron3_1.py

```diff
@@ -30,7 +54,8 @@ def test_coron3_pipeline1(_bigdata):
     newh = pf.HDUList([h['primary'], h['sci'], h['err'], h['dq']])
     newhref = pf.HDUList([href['primary'], href['sci'],
                          href['err'], href['dq']])
-    kws_to_ignore = ['DATE', 'CAL_VER', 'CAL_VCS', 'CRDS_VER', 'CRDS_CTX']
+    kws_to_ignore = ['DATE', 'CAL_VER', 'CAL_VCS', 'CRDS_VER', 'CRDS_CTX', 
+                     'R_PSFMAS']
```

Comment:
I still don't like this idea of ignoring the reference file name keywords. When a new ref file gets installed in CRDS and the test starts showing differences in the results, you won't know why (because it won't tell you that the ref file changed). You'll have to do more detailed sleuthing on your own to figure out why things suddenly changed. Once you get a clean run and the only difference is the name of ref file, just update the comparison file so that it has that new ref file name.

Reply:
The best solution would be to not include the absolute path when recording the name of this specific filename in the metadata/keyword in the first place.  This would actually require a change to the code itself which is beyond the scope of these changes.  Until then, I can back out this change.